pallets/asset-metadata-extender/src/precompiles/mod.rs

```diff
@@ -15,6 +15,6 @@ pub fn register_cost<Runtime: crate::Config>(
 		return Err(ExitError::OutOfGas);
 	}
 	handle.record_cost(required_gas)?;
-	handle.record_external_cost(Some(weight.ref_time()), Some(weight.proof_size()))?;
+	handle.record_external_cost(Some(weight.ref_time()), Some(weight.proof_size()), Some(0))?; // TODO duplicated fn
```

Comment:
TODO

Reply:
This is a refactor that will be done after the current PR is merged. So that we ensure to have the functionality as soon as possible and leave the refactor for later.

Reply:
is it possible to add some more infos ? I was unable to understand it when I red it.